apps/dashboard/src/components/icons/inbox-bell-filled.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export function InboxBellFilled(props: React.ComponentPropsWithoutRef<'svg'>) {
```

Comment:
Didn't even know this existed. Why not `HTMLAttributes<HTMLSvgElement>` ?

Reply:
Just copied from elsewhere, seems there's a bit of a mixture throughout the icons folder. I updated to your suggestion, it's more pure and accurate for this case.